internal/runners/initialize/init.go

```diff
@@ -189,10 +189,10 @@ func (r *Initialize) Run(params *RunParams) (rerr error) {
 
 	version, err := deriveVersion(lang, languageVersion, r.auth)
 	if err != nil {
-		if inferred || !locale.IsInputError(err) {
+		if inferred || (!locale.IsInputError(err) && !errs.IsExternalError(err)) {
```

Comment:
This is at least the second time I've seen this. Would it make sense to add a function that would wrap this so if we were to add more types of errors, we'd only have to change that function instead of all of these instances?

Reply:
This is a good idea. Thanks for the suggestion.